src/zeek/network.rs

```diff
@@ -318,6 +319,7 @@ impl TryFromZeekRecord for Dns {
                 resp_addr,
                 resp_port,
                 proto,
+                start_time: time,
                 end_time: rtt,
```

Comment:
The `end_time` value must be set to:
  `end_time = time + rtt`

Reply:
done